src/components/ChecAccordion.vue

```diff
@@ -4,26 +4,28 @@
     class="accordion"
     :class="{'accordion--active': isOpen}"
   >
-    <div class="accordion__heading">
+    <div class="accordion__heading" @click="handleHeadingClick">
```

Comment:
Do you need to give it a role attribute for keyboard accessibility?

Reply:
Nope. The keyboard should still highlight the actual switch or button, I don't think we want the header to be tab-navable.

Reply:
Alternatively I can make the header/accordion tabbable, and make the button not tabbable - but that's a bunch more work